content/Certification/certifying-a-product/development.md

```diff
@@ -0,0 +1,200 @@
++++
+title = "Development"
+chapter = false
+weight = 6
++++
+
+![](../imgs/development.png)
+
+The process of developing a Matter product is highly dependent on the device
+being developed, the platform being used, and the policies of the company doing the development. There
+is no single "best" way to integrate Matter into a product. This guide aims to
+address the common considerations for Matter product development.
+
+- Information on the Connectivity Standards Alliance (CSA) open source SDK is available at the
+[Matter SDK documentation site](https://project-chip.github.io/connectedhomeip-doc/index.html).
+
+- Integration of the SDK on specific platforms is covered by the
+[Platform guides](https://project-chip.github.io/connectedhomeip-doc/platforms/index.html).
+
+
+## Matter-specific product considerations
+The documentation in this section aims to give a brief overview of some of the
```

Comment:
For readability in the source, can you add a blank line after each header in these markdown pages?  Some have it and some don't.